src/Microsoft.VisualStudio.Services.Agent/Util/ArgUtil.cs

```diff
@@ -4,6 +4,23 @@ namespace Microsoft.VisualStudio.Services.Agent.Util
 {
     public static class ArgUtil
     {
+        public static void Equal<T>(T expected, T actual, string name)
```

Comment:
It's been many years since I've read `50 tips to improve your C#` (good book), but I think this suffices for determining equality :)